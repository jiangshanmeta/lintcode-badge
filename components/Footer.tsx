import { Box,  Link, Typography } from "@material-ui/core";
import { FC } from "react";

const Footer: FC = () => (
    <footer>
        <Box
            display="flex"
            flexDirection="column"
            alignItems="center"
            justifyContent="center"
        >
            <Typography variant="caption" align="center">
        Built on top of <Link href="https://lintcode.com/">LintCode.com</Link>{" "}
        and <Link href="https://shields.io/">Shields.io</Link>
                <br />
        by <Link href="https://github.com/jiangshanmeta">jiangshanmeta</Link>
            </Typography>
        </Box>
    </footer>
);

export default Footer;
