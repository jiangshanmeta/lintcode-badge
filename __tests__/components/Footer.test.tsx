import {
    render, 
    screen,
} from '@testing-library/react';
import '@testing-library/jest-dom';

import Footer from 'components/Footer'

describe('Component Footer',()=>{
    it('should match snapshot',()=>{
        const {
            container,
        } = render(<Footer/>);

        expect(container.innerHTML).toMatchSnapshot()
    })
})