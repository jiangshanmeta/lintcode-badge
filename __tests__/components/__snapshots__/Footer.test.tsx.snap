// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Footer should match snapshot 1`] = `"<footer><div class="MuiBox-root MuiBox-root-1"><span class="MuiTypography-root MuiTypography-caption MuiTypography-alignCenter">Built on top of <a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary" href="https://lintcode.com/">LintCode.com</a> and <a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary" href="https://shields.io/">Shields.io</a><br>by <a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary" href="https://github.com/jiangshanmeta">jiangshanmeta</a></span></div></footer>"`;
